core/src/main/java/io/questdb/griffin/engine/functions/catalogue/PgIndexFunctionFactory.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2023 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.griffin.engine.functions.catalogue;
+
+import io.questdb.cairo.ColumnType;
+import io.questdb.cairo.GenericRecordMetadata;
+import io.questdb.cairo.TableColumnMetadata;
+import io.questdb.cairo.sql.RecordMetadata;
+
+public class PgIndexFunctionFactory extends AbstractEmptyCatalogueFunctionFactory {
+    private final static RecordMetadata METADATA;
+
+    public PgIndexFunctionFactory() {
+        this("pg_index()");
+    }
+
+    protected PgIndexFunctionFactory(String signature) {
+        super(signature, METADATA);
+    }
+
+    @Override
+    public boolean isRuntimeConstant() {
+        return true;
+    }
+
```

Comment:
Wouldn't it be better to include all columns ?
https://www.postgresql.org/docs/current/catalog-pg-index.html

Reply:
This one is also old. Added the missing columns in bcf6180c0d58a1e021fc3ee5f9b46b4101bf63b2